### DISCUSSION TITLE
Touch Screen Login, Keyboard required or simple PIN with Num Pad

### DISCUSSION THREAD

**Discussion by RobertBudgen** [frangoteam/FUXA#931] [Q&A]
Is it possible to enable the onscreen keyboard for Login.
Our system will be running on a touch screen in  Kiosk Mode, no Physical keyboard  or Windows keyboard  is available.

**Comment by TomasAdvance** [comment]
Hi Fuxa Team,
it will be very helpfull if  onscreen keyboard will be availabe for user Login
BR
T
